### DISCUSSION TITLE
How to create a C# Action object

### DISCUSSION THREAD

**Discussion by qingpengchen2011** [vfsfitvnm/frida-il2cpp-bridge#176] [Q&A]
Here is the C# function I want to make a call:
 SFSRequestWaiter.Instance.WaitRequest("login", new Action<SFSObject>(this.callbackfunction));

I have  2 questions:
1. How to create [new Action<SFSObject>] by frida.
2. if the address of this.callbackfunction is Method.virtualAddress.

**Comment by vfsfitvnm** [comment]
Probably #153 may help you

**Reply by qingpengchen2011** [reply]
Nice~
